app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
     pass
```

Comment:
this requirement isn't met:
This class also should have a class attribute people, it is a dict that stores Person instances by their name. The __init__ method should add elements to this attribute.